### ISSUE TITLE
[Question] Is there a way to assign worker per directory?

### ISSUE THREAD

**Issue by ktakdev** [microsoft/playwright#17495]
I want workers to test basically in parallel, but sequentially in a directory.
e.g. worker1 executes tests in directory1, worker2 executes tests in directory2...

Currently, workers are automatically assigned per file in default,
that means both worker1 and worker2 execute tests in directory1 in parallel, then directory2.

Can any tell me how to assign worker per directory if the way exists.

**Comment by Sam-H101** [comment]
This may be related, if not please ignore. 

Granted this is python, but this is how I am handling parallelism with remote execution. 

https://github.com/Sam-H101/playwright-remote-orchestrator

All that is due to my strong desire to avoid selenium grid like the plague.

**Comment by pavelfeldman** [comment]
#### Why was this issue closed?

Thank you for your involvement. This issue was closed due to limited engagement (upvotes/activity), lack of recent activity, and insufficient actionability. To maintain a manageable database, we prioritize issues based on these factors.

If you disagree with this closure, please open a new issue and reference this one. More support or clarity on its necessity may prompt a review. Your understanding and cooperation are appreciated.
